core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -120,11 +177,16 @@ public void testScenario_010() throws Exception {
       rpcManager0.blockBefore(ClusteredGetCommand.class);
       topologyManager0.startBlocking(LatchType.REBALANCE);
 
+      cache(0).getAdvancedCache().getAsyncInterceptorChain()
+            .addInterceptorAfter(new AssertNoRetryInterceptor(), StateTransferInterceptor.class);
+
       //remote get is processed in current topology id.
       Future<Object> remoteGetFuture = remoteGet(cache(0), key);
       rpcManager0.waitForCommandToBlock();
 
-      NewNode joiner = addNode();
+      NewNode joiner = addNode(null, cb -> cb.customInterceptors().addInterceptor()
+         .position(InterceptorConfiguration.Position.FIRST).interceptor(new FailReadsInterceptor()));
```

Comment:
IMO it's not really an assertion because the exception gets ignored if there's a successful response from another node, so I would have preferred an out-of-band way to assert there was no request to this node. (e.g. with an `assertNoHits` method on the interceptor).